docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/global-default-private-registry.md

```diff
@@ -54,8 +54,20 @@ Since the private registry cannot be configured after the cluster is created, yo
 1. Select **☰ > Cluster Management**.
 1. On the **Clusters** page, click **Create**.
 1. Choose a cluster type.
-1. In the **Cluster Configuration** go to the **Registries** tab and select **Pull images for Rancher from a private registry**.
-1. Enter the registry hostname and credentials.
+1. In the **Cluster Configuration** go to the **Registries** tab.
+1. Check the box next to **Enable cluster scoped container registry for Rancher system container images**.
+1. Enter the registry hostname.
+1. Under **Authentication** select **Create a HTTP Basic Auth Secret** and fill in the credential fields.
 1. Click **Create**.
 
 **Result:** The new cluster pulls images from the private registry.
+
+### Working with Private Registry Credentials
+
+When working with private registries, it is important to ensure that any secrets created for these registries are properly backed up. By default, when you add a private registry credential secret through the process outlined above, it is included in backup operations using Rancher Backups.
```

Comment:
So the specificity here has to clarify that if a user does it via the GUI it could still be missed in backups. Since the real key making it work (without the label) is selecting "Create a HTTP Basic Auth Secret" from the Cluster creation/update page.

Where as a user could, all from the GUI still, go create a manual secret in the `fleet-default` namespace that doesn't conform to the name convention used when selecting "Create a..." and will need the label added by the user if they select that secret from the drop down box.

Reply:
I'll edit the suggestion to reflect this